worlds/kh1/Items.py

```diff
@@ -0,0 +1,534 @@
+   #"Blaze Shard":                       KH1ItemData("Synthesis",        code = 264_1241, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+   #"Blaze Gem":                         KH1ItemData("Synthesis",        code = 264_1242, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+   #"Frost Shard":                       KH1ItemData("Synthesis",        code = 264_1243, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+   #"Frost Gem":                         KH1ItemData("Synthesis",        code = 264_1244, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+   #"Thunder Shard":                     KH1ItemData("Synthesis",        code = 264_1245, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+   #"Thunder Gem":                       KH1ItemData("Synthesis",        code = 264_1246, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+   #"Shiny Crystal":                     KH1ItemData("Synthesis",        code = 264_1247, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+   #"Bright Shard":                      KH1ItemData("Synthesis",        code = 264_1248, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+   #"Bright Gem":                        KH1ItemData("Synthesis",        code = 264_1249, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+   #"Bright Crystal":                    KH1ItemData("Synthesis",        code = 264_1250, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+   #"Mystery Goo":                       KH1ItemData("Synthesis",        code = 264_1251, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+   #"Gale":                              KH1ItemData("Synthesis",        code = 264_1252, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+   #"Mythril Shard":                     KH1ItemData("Synthesis",        code = 264_1253, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+   #"Mythril":                           KH1ItemData("Synthesis",        code = 264_1254, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+   #"Orichalcum":                        KH1ItemData("Synthesis",        code = 264_1255, classification = ItemClassification.filler,      max_quantity = 1,  weight = 10),
+    "High Jump":                         KH1ItemData("Shared Abilities", code = 264_2001, classification = ItemClassification.progression, max_quantity = 1,  weight = 10),
+    "Mermaid Kick":                      KH1ItemData("Atlantica"       , code = 264_2002, classification = ItemClassification.progression, max_quantity = 1,  weight = 10),
```

Comment:
The item group here was probably meant to be Shared Abilities

Reply:
Mermaid Kick should only be in the pool if Atlantica is on, hence the different item group.